action/create_vm.go

```diff
@@ -47,20 +52,33 @@ func (a CreateVM) Run(agentID string, stemcellCID StemcellCID, cloudProps bslcvm
 	return VMCID(vm.ID()).String(), nil
 }
 
-func (a CreateVM) updateCloudProperties(cloudProps bslcvm.VMCloudProperties) {
-	if cloudProps.StartCpus > 1 {
-		a.vmCloudProperties.StartCpus = cloudProps.StartCpus
+func (a CreateVM) UpdateCloudProperties(cloudProps *bslcvm.VMCloudProperties) {
```

Comment:
@maximilien  Since it is a change related with existing function, just implementation approach changed and the test cases had been changed accordingly, so no additional test cases needed.